TomoPool.sol

```diff
@@ -424,6 +431,12 @@ contract CandidateContract {
         return governance.supportCap >= capacity.mul(66).div(100);
     }
 
+    function resignIfUnder60k() {
```

Comment:
just concern that it should be onlyOwner or onlyStaker.
It will be an issuer if the network is not full 150 MNs.

Reply:
agree! it should be staker only